app/src/main/java/org/simple/clinic/scanid/PatientPrefillInfo.kt

```diff
@@ -0,0 +1,32 @@
+package org.simple.clinic.scanid
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class PatientPrefillInfo(
```

Comment:
Aren't these variables supposed to be nullable?

Reply:
We are getting all the fields from the NHID. Yes, making state and district to be nullable.

Reply:
Have checked the documentation to make sure the rest of the fields won't be nullable in any scenario?

Reply:
Yes i have